lib/std/tz.zig

```diff
@@ -0,0 +1,233 @@
+const std = @import("std.zig");
+const builtin = @import("builtin");
+
+pub const Transition = struct {
+    ts: i64,
+    timetype: *Timetype,
+};
+
+pub const Timetype = struct {
+    offset: i32,
+    flags: u8,
```

Comment:
Why have this as `flags` instead of `bool` fields for each?

Reply:
It makes `Timetype` pack better; 12 bytes instead of 16. I can change it if the optimization isn't worth the complexity of having getters for each value.